src/main/java/com/aviva/CarRecommendations/EstimateBudget.java

```diff
@@ -1,4 +1,26 @@
 package com.aviva.CarRecommendations;
 
+import com.aviva.DataProcess.BankingDataProcess;
+
+import java.util.ArrayList;
+
 public class EstimateBudget {
+    public static float calculateYearlyBudget(String accountNumber) {
```

Comment:
This needs documentation

Reply:
We will add documentation to everything at the end we just want to see something functional atm.